### PR TITLE
Update security-considerations-best-practices.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

**Comment by ujpat** [comment]
@bluefooted and @MashaMSFT  for visibility

**Comment by Court72** [comment]
@bluefooted 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by bluefooted — APPROVED** [review]

**Comment by bluefooted** [comment]
#sign-off

File: azure-sql/virtual-machines/windows/security-considerations-best-practices.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Consider the following limitations
 - Once Azure AD authentication is enabled, there is no way to disable it by using the Azure portal. 
 - Currently, enabling Azure AD authentication is only possible through the Azure portal. 
 - Currently, Azure AD authentication is only available to SQL Server VMs deployed to the public cloud. 
-
+- Currently, Authenticating to SQL VM through AAD authentication using [FIDO2 method](/azure/active-directory/authentication/howto-authentication-passwordless-faqs.md) is not supported.
```
Comment (Jak-MS) [inline azure-sql/virtual-machines/windows/security-considerations-best-practices.md]:
```suggestion
- Currently, Authenticating to SQL VM through AAD authentication using [FIDO2 method](/azure/active-directory/authentication/howto-authentication-passwordless-faqs) is not supported.
```

**Pull request merged**